content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/index.md

```diff
@@ -8,7 +8,8 @@ topics:
 children:
   - /enabling-github-actions-with-azure-blob-storage
   - /enabling-github-actions-with-amazon-s3-storage
-  - /enabling-github-actions-with-minio-gateway-for-nas-storage
+  - /enabling-github-actions-with-minio-storage
```

Comment:
Since you need to configure your storage bucket before enabling Actions, let's reverse the order of these.